qa/tasks/cephfs/test_volumes.py

```diff
@@ -5833,7 +5833,7 @@ def test_subvolume_snapshot_info_if_orphan_clone(self):
 
         # remove track file for third clone to make it orphan
         meta_path = os.path.join(".", "volumes", "_nogroup", subvolume, ".meta")
-        pending_clones_result = self.mount_a.run_shell(f"sudo grep \"clone snaps\" -A3 {meta_path}", omit_sudo=False, stdout=StringIO(), stderr=StringIO())
+        pending_clones_result = self.mount_a.run_shell(f"sudo grep clone\ snaps -A3 {meta_path}", omit_sudo=False, stdout=StringIO(), stderr=StringIO())
```

Comment:
@kotreshhr was this test always failing?

Reply:
Nope, only on ubuntu lab machines. It passes on ubuntu on vstart cluster locally though.

Reply:
it's strange grep yields different results in vstart_runner vs lab. BTW is the local environment using different version(i.e. other than focal)?